careerfairsystems/arkad-app-2018
# Create favorites button

## Change description

Adds the Font Awesome icon library to the project. Icons are added for the favorites button and the tab navigator.

## How to verify

Click on the heart and the different tabs (see how the color changes) (arkad icon isn't working correctly, see issue #39 )

## Issues fixed

Fixes #36 and #13
